### PR TITLE
[CI] add vllm tests

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Add VLLM tests for rolling batch

### CODE REVIEW

File: tests/integration/launch_container.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ if [[ "$platform" == *"cu1"* ]]; then # if the platform has cuda capabilities
 elif [[ "$platform" == *"deepspeed"* ]]; then # Runs multi-gpu
   runtime="nvidia"
   is_llm=true
-  shm="2gb"
+  shm="12gb"
```
Comment (frankfliu) [inline tests/integration/launch_container.sh:29]: Why need 12gb? Does SM support this size?

Comment (lanking520) [reply tests/integration/launch_container.sh:29]: vLLM test requires minimum 10GB shared memory spaces

Comment (lanking520) [reply tests/integration/launch_container.sh:29]: for the best performance

**Comment by lanking520** [comment]
test pased: https://github.com/deepjavalibrary/djl-serving/actions/runs/6151354963/job/16691281183

**Review by frankfliu — APPROVED** [review]

**Pull request merged**